e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -181,8 +181,11 @@ test.describe('Example Imagery Object', () => {
 
     });
 
-    test('Can use the reset button to reset the image', async ({ page }, testInfo) => {
-        test.slow(testInfo.project === 'chrome-beta', "This test is slow in chrome-beta");
+    test('Can use the reset button to reset the image', async ({ page, browser }, testInfo) => {
+        test.slow(
+            testInfo.project === 'chrome-beta' && browser.version() === '104.0.5112.39',
```

Comment:
In this case, I think we'd want to check the version of chrome. It's possible that this performance regression will not be fixed and become mainline chrome.

I would set this test to slow if brower.version() > 103 && browser='chrome'